tsk/vs/gpt.c

```diff
@@ -16,6 +16,73 @@
 #include "tsk_dos.h"
 
 
+/*
+ *Check if GUID matches a given value.
+ */
```

Comment:
Can you add basic doxygen-style API docs.

/**
 * check if GUID matches a given value.
 * @param g GUID to match
* @param d1 ....
....
* @returns 1 if they match, 0 if they do not
*/